src/Google.Cloud.Functions.Invoker/EntryPoint.cs

```diff
@@ -60,7 +60,14 @@ public static async Task<int> StartAsync(Assembly functionAssembly, string[] arg
             // doesn't matter much. Potentially catch exceptions during configuration, but let any
             // during web server execution propagate.
             var host = Host.CreateDefaultBuilder()
-                .ConfigureWebHostDefaults(webHostBuilder => webHostBuilder.UseFunctionsFramework(functionAssembly, args))
+                .ConfigureWebHostDefaults(webHostBuilder => webHostBuilder
+                    .ConfigureAppConfiguration(builder => builder.AddFunctionsEnvironment().AddFunctionsCommandLine(args))
+                    .ConfigureLogging((context, logging) => logging.ClearProviders().AddFunctionsConsoleLogging(context))
+                    .ConfigureKestrelForFunctionsFramework()
+                    .ConfigureServices((context, services) => services
+                    .AddFunctionTarget(context, functionAssembly))
```

Comment:
Consider creating our own DefaultStartup and adding all of these in there (except maybe the Kestrel bit) and then calling here .UseFunctionStartup(new DefaultStartup()) just before .UserFunctionsStartups(functionAssembly).

Reply:
Maybe. That has a certain elegance to it, but it possibly obscures what we're doing a little. Let's leave that as an idea for the future.